ogame/test.py

```diff
@@ -174,6 +174,20 @@ def test_build(self):
         ).rocket_launcher
         self.assertTrue(before < after.amount or after.in_construction)
 
+    def test_deconstruct(self):
```

Comment:
is it possible to use somthing mich cheaper to deconstrukt or at least cancel it. would produce unwanted side effects. losing my 10 mio metal mine when i test my code

Reply:
Yeah... I was thinking about that too. But I don't know what to use instead. Any advice?
I was doing my tests with _buildings.missile_silo_ on the only planet where I have it at level 1

Also, I noticed now that I'm calling a tuple on line 183: `what=buildings.metal_mine(),`. Going to fix that..